cmd/amps/broker/amqp.go

```diff
@@ -29,19 +44,29 @@ type AMQPBroker struct {
 	consumeChannel       *amqp.Channel
 	publishChannel       *amqp.Channel
 	connMutex            *sync.RWMutex
-	shutdownChan         chan bool
-	isShuttingDown       bool
+
+	// FIX: Removed 'isShuttingDown' bool — context now handles safe shutdown signaling
```

Comment:
Might want to remove this line before merging

Reply:
done!